### PR TITLE
[yaml/zh-cn] retranslate some content

### PR DESCRIPTION
**Pull request by HermitSun**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

---

以下是译注，以供reviewer参考：

1. 原文为'syntactically significant newlines and indentation'，原译为“语法显著换行符和缩进”。
   个人认为，significant在此处应为“有意义的”，而非“显著的”。联系上下文，此处应指YAML的换行和缩进具有语法含义，而非在语法上显著（虽然确实很醒目）。
   故译为“在语法上有意义的换行符和缩进”。

2. 原文为'UTF-8/16/32 characters need to be encoded'，原译为“UTF-8/16/32 字符需要被转义”。个人认为，“转义”应为escape，并且虽然`\u`本身是一个转义字符，但目的是为了表明编码。
   故译为“UTF-8/16/32字符需要指明编码”。

3. “字面量块”即'literal block'，此处取literal的“字面”之意。
   从下文也可以看到，字面量块会保留原字符串，包括换行等特殊字符；而折叠块会把原字符串的每一行用空格分隔，然后合并到同一行。

4. 原文为'The regexp << is called Merge Key Language-Independent Type.'。若regexp直译为正则表达式，似有不妥，故在尽量保留含义的基础上有所省略。

### CODE REVIEW

**Comment by imba-tjd** [comment]
I disagree about removing the spaces between en and zh words.

**Comment by HermitSun** [comment]
> I disagree about removing the spaces between en and zh words.

Ah, If so, I will revert it.

File: zh-cn/yaml-cn.html.markdown [hunk 1]
```diff
@@ -77,22 +77,22 @@ a_nested_map:
...
+# 但有些解析器可能会不支持。
 # 一个例子：
 ? - Manchester United
   - Real Madrid
 : [ 2001-01-01, 2002-02-02 ]
 
-# 序列 (等价于列表或数组) 看起来像这样：
-# 注意 '-' 算作缩进
+# 序列 (sequences，等价于列表list或数组array) 看起来像这样
+# 注意 '-' 代表缩进：
```
Comment (imba-tjd) [inline zh-cn/yaml-cn.html.markdown:95]: 这里我个人觉得翻译成“也算缩进”比较好

Comment (HermitSun) [reply zh-cn/yaml-cn.html.markdown:95]: 有道理，我改一下。

Comment (imba-tjd) [reply zh-cn/yaml-cn.html.markdown:95]: 没了:joy:

Comment (HermitSun) [reply zh-cn/yaml-cn.html.markdown]: 好了

Comment (imba-tjd) [reply zh-cn/yaml-cn.html.markdown]: 关于中文文章的排版还可以看 https://github.com/sparanoid/chinese-copywriting-guidelines
当然，那只是一些建议。我自己的博客中英文之间也是不加空格的。

Comment (HermitSun) [reply zh-cn/yaml-cn.html.markdown]: 感谢。我自己写的时候喜欢用反引号😂

**Comment by kevinnls** [comment]
@mxschumacher

**Comment by mxschumacher** [comment]
Thank you for the contribution, @HermitSun !!

**Pull request merged**